library/src/app/components/external-wallet/external-wallet-create/external-wallet-create.component.scss

```diff
@@ -0,0 +1,49 @@
+.cybrid-header {
```

Comment:
Aren't some of these available as global css classes? They don't necessarily have to be repeated unless you've made changes.